code/modules/admin/panicbunker.dm

```diff
@@ -0,0 +1,37 @@
+var/list/panic_bunker_bypass = list()
+
+/datum/admins/proc/panicbunker()
+	set category = "Server"
+	set name = "Toggle Panic Bunker"
+
+	if(!check_rights(R_SERVER))
+		return
+
+	config.panic_bunker = !config.panic_bunker
+
+	log_and_message_admins("[key_name(usr)] has toggled the Panic Bunker, it is now [(config.panic_bunker ? "on" : "off")]")
```

Comment:
log_and_message_admins("has toggled the Panic Bunker, it is now [(config.panic_bunker ? "on" : "off")]"`